compiler/rustc_lint_defs/src/builtin.rs

```diff
@@ -3339,11 +3339,12 @@ declare_lint! {
     ///
     /// ### Explanation
     ///
-    /// This lint is only active when `--check-cfg` arguments are being passed
-    /// to the compiler and triggers whenever an unexpected condition name or value is used.
+    /// This lint is only active when [`--check-cfg`](../../check-cfg.md) arguments are being
```

Comment:
This relative path feels weird to me because it's not relative to the current file. Do we sth. similar in other places? I'd go for an absolute path / URL.

I'm pretty sure that this link is broken outside of *The rustc book*, namely over at https://doc.rust-lang.org/nightly/nightly-rustc / when documented with `./x doc compiler`

Reply:
I wanted to avoid the absolute path so that stable users get the stable doc but it doesn't seems worth breaking any other use case. We even also use the nightly path inside the diagnostic output so changed it to be absolute.